### DISCUSSION TITLE
access to one page data from another page

### DISCUSSION THREAD

**Discussion by alephpi** [vuejs/vitepress#2459] [Q&A]
Hi, I want to have a index page that lists necessary information of a collection of pages.
```
/src
|-index.md
|-1.md
|-2.md
```
I can already load information e.g. frontmatter via build-time data loading. However, I also want to access to last updated date of 1/2.md from index.md. We can use  runtime API but it seems only working in the page itself.

**Comment by zonemeen** [comment]
I think this might be helpful. https://vitepress.dev/guide/data-loading#createcontentloader

**Reply by alephpi** [reply]
this is what I did for the moment, however it seems I have to manually set date in frontmatter and I wonder if there are any method to get the updated time automatically as in runtime API.
